### PR TITLE
style cleanup

### LINKED ISSUE #72: Does the UI look correct
- [ ] drag box should be as generous as other controls
- [ ] rounded corners on the play head

### PR DESCRIPTION
**Pull request by johngruen**
Fixes #72

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "name": "min-vid",
   "version": "0.1.9",
   "private": true,
-  "icon": "resource://min-vid/docs/images/gradient-logo.png",
+  "icon": "resource://min-vid/docs/images/dark-logo.png",
```
Comment (johngruen) [inline package.json]: hey @meandavejustice for some reason this icon doesn't show up in about:debugging or about:addons

Comment (johngruen) [inline package.json]: thoughts?

Comment (pdehaan) [inline package.json]: It isn't in the bundled XPI:

index_of_jar_file____var_folders_qc_g_f0szh95k1fzph34rhmf5jw0000gp_t_f8476a30-a1ee-4ee7-b6a7-7b90e7dff2e3_extensions__min-vid_xpi__docs_images_

Comment (pdehaan) [inline package.json]: add-ons_manager

👍

Comment (johngruen) [inline package.json]: woot

**Pull request merged**